docs: improve note with latest format (#4493)
webpack/webpack-dev-server: README.md

Made with ♥ by the webpack team.
```

> **Note**
>

> > _For more information on above options explore this [link](https://webpack.js.org/configuration/dev-server/)._

Reviewer: ```suggestion
> _Detailed documentation for above options is available on this [link](https://webpack.js.org/configuration/dev-server/)._
```
